### DISCUSSION TITLE
Configuration of Detekt

### DISCUSSION THREAD

**Discussion by TimGels** [TimGels/Camera-Security-System-Client#25] [General]
This discussion will discuss the configuration of our Detekt workflow. Wanted changes can be discussed here and I will implement them when agreed upon.

@JochemFB @JohannesKauffmann @SietzeKoonstra

**Comment by JohannesKauffmann** [comment]
Currently, kdetect complains about TooGenericException (basically `catch(ex: Exception)` is not allowed). However, I'd argue that having SwallowedException is far more valuable, since then, even when catching generic exceptions, at least they are logged somewhere.

**Reply by TimGels** [reply]
I agree, I will disable `TooManyFunctions`, `TooGenericExceptionCaught` and remove `TODO` from the `ForbiddenComment`.
